orderapi/src/main/resources/application.properties

```diff
@@ -32,10 +32,13 @@ subscription.renewal.batch.sleep.millis=10000
 subscription.notification.expiring.card.threshold.days=7
 
 serviceconfiguration.url=http://product-mapping-api:8080/serviceconfiguration/
+message.service.url=http://message-api:8080/message/
 
 order.experience.url=http://order-experience-api:8080/v1/order/
 
 spring.activemq.user=
 spring.activemq.password=
 
-spring.devtools.restart.enabled=false
\ No newline at end of file
+spring.devtools.restart.enabled=false
+
+payment.failed.notification.email=${PAYMENT_FAILED_NOTIFICATION_EMAIL}
```

Comment:
You can just add the default email here, no need to get it from env variable when using local profile.